domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -283,6 +283,30 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+  /**
+   * Marks one story progress full in ratios exploration for a particular profile.
+   *
+   * @param profileId the profile we are setting topic progress on ratios for
+   * @param timestampOlderThanOneWeek if the timestamp for this progress is from more than one week ago
+   */
+  fun markChapDoneOfRatiosStory0Exp2(
```

Comment:
Is this name correct? It seems to be marking exploration 1 done, not 2.

Reply:
Done.